service/common/src/main/java/org/apache/polaris/service/events/EventListenerConfiguration.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.polaris.service.events;
+
+import java.util.Optional;
+
+public interface EventListenerConfiguration {
+  Optional<String> awsCloudwatchlogGroup();
+
+  Optional<String> awsCloudwatchlogStream();
+
+  Optional<String> awsCloudwatchRegion();
```

Comment:
How does this work if I want to define my own EventListener -- do I essentially need to fork and modify this class?

Won't this also grow quite large as more listeners get implemented?

Reply:
> How does this work if I want to define my own EventListener -- do I essentially need to fork and modify this class?

That's one way to do it. The other would be to create a new interface and have `QuarkusPolarisEventListenerConfiguration` extend the new interface as well. Maybe there's a different way to do it, but I haven't found one yet.

> Won't this also grow quite large as more listeners get implemented?

It could. If we start getting a lot of new listeners, we can always split into multiple interfaces. I'm not doing this yet (it might be premature optimization) but we could very easily do this in the future.

Reply:
If possible, I think it would make sense to create a different configuration for each listener -- ideally, in such a way to make it easy for users to add their own listener. This should also help isolate the code of various listeners.